quartz/plugins/transformers/lastmod.ts

```diff
@@ -56,11 +57,14 @@ export const CreatedModifiedDate: QuartzTransformerPlugin<Partial<Options>> = (u
                   // Get a reference to the main git repo.
                   // It's either the same as the workdir,
                   // or 1+ level higher in case of a submodule/subtree setup
-                  repo = Repository.discover(file.cwd)
+                  repo = Repository.discover(fp)
+                  repoRoot = path.join(repo.path(), "..")
```

Comment:
This seems to assume that the other repository is next to the Quartz repository. Does this still work for any arbitrary repository path?

Reply:
`repo.path()` returns a path to the `.git` directory _inside_ the repo. I was confused by this too, and then saw from their examples that this is just how it works:

https://github.com/Brooooooklyn/simple-git/blob/67d0983f610c348c9d860a42fa9d3c8fd7d408a4/example.mjs#L9-L11